lib/raidenclient/RaidenClient.ts

```diff
@@ -472,6 +482,47 @@ class RaidenClient extends SwapClient {
     return { balance, pendingOpenBalance: 0, inactiveBalance: 0 };
   }
 
+  public tradingLimits = async (currency?: string): Promise<TradingLimits> => {
+    if (!currency) {
+      return { maxSell: 0, maxBuy: 0 };
+    }
+
+    const channels = await this.getChannels(this.tokenAddresses.get(currency));
+
+    let maxOutbound = 0;
+    let maxInbound = 0;
+    channels.forEach((channel) => {
+      if (channel.state !== 'open') {
+        return;
+      }
+
+      const outbound = channel.balance;
+      if (maxOutbound < outbound) {
+        maxOutbound = outbound;
+      }
+
+      const inbound = channel.total_deposit - channel.balance;
+      if (maxInbound < inbound) {
+        maxInbound = inbound;
+      }
+    });
+
```

Comment:
I think an important step here would be to convert the max channel balances we've fetched from the base "units" for each token, which is 10^-18 for the ERC20 tokens we're using so far, to the "amounts" that we use throughout the codebase for orders which are represented with satoshis/10^-8.